assets/constants.js

```diff
@@ -464,7 +464,7 @@ var constants = {
     "\\u001b", "\\u001c", "\\u001d", "\\u001e", "\\u001f"
   ],
 
-  "ENABLE_NEW_STRUCTURE_EDITORS": false,
+  "ENABLE_NEW_STRUCTURE_EDITORS": true,
```

Comment:
Do we want this flipped in develop?

Reply:
Yeah, since we are releasing it next month, this has to be enabled anyway. So, thought it could be flipped in this PR itself as e2e tests are also being created now.

Reply:
Oh ok. :)